blockly/msg/json/nl.json

```diff
@@ -7,7 +7,8 @@
 			"MedShot",
 			"아라",
 			"JaapDeKleine",
-			"Sjoerddebruin"
+			"Sjoerddebruin",
+			"Benny Malengier"
```

Comment:
Yeah, hopefully it should be a straightforward merge conflict the first time the authors get updated in the wiki.